### PR TITLE
Ogg Opus encapsulation for voice listening

### PR DESCRIPTION
**Pull request by BrandtHill**
Initial pass at adding functionality for listening on voice channels

### CODE REVIEW

**Review by jchristgit — APPROVED** [review]
Rock solid. Thanks!

File: lib/nostrum/voice/opus.ex [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
+      comments: comments,
+      rest: rest
+    }
+  end
+
+  def encode_opus_tags(tags) do
+    %{
+      vendor_string: vendor_string,
+      comments: comments
+    } = tags
```
[the remark this answers is not in the document]
Comment (BrandtHill) [reply lib/nostrum/voice/opus.ex]: done

[the remark this answers is not in the document]
Comment (jchristgit) [reply lib/nostrum/voice/opus.ex]: winner

**Comment by jchristgit** [comment]
Thanks!

**Pull request merged**